### DISCUSSION TITLE
Monthly ESMValTool community meeting - March 2024

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#155] [Meetings]
The next online community meeting will be on 

28th March at 1300 GMT.

The duration will be one hour.  [Join the meeting](https://ncas.zoom.us/j/97855095315?pwd=WVkwSlIxbFhOK0VIR3RKVmtzeXd1QT09 )

Calendar invitations to the meeting are sent to the mailing list, join it [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

**Format and agenda**

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #148

**Comment by paultgriffiths** [comment]
Updates
@glpotter - will start a new GH ESMValTool Discussion around the technical challenges of CMOR-ising reanalyses (and the scientific opportunities that would arise from having these data)
@ehogan CMEW nearing a stable release.  Will form a nice cylc pipeline, interfacing with CDDS, to produce standardised UKESM output.

**Reply by glpotter** [reply]
@paultgriffiths I have not yet entered a new discussion, I am waiting for news from the Copernicus folks about their interest in cmorizing reanalysis..
